### PR TITLE
chore: update license + improve README

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2022 the denosaurs team
+Copyright (c) 2023 the denosaurs team
```
Comment (eliassjogreen) [inline LICENSE:3]:
```suggestion
Copyright (c) 2022-2023 the denosaurs team
```

File: README.md [hunk 2]
```diff
@@ -17,17 +17,19 @@ await serve(
...
-
 ### Contribution
 
 Pull request, issues and feedback are very welcome. Code style is formatted with
 `deno fmt` and commit messages are done following Conventional Commits spec.
 
 ### Licence
 
-Copyright 2022, the denosaurs team. All rights reserved. MIT license.
+Copyright 2023, the denosaurs team. All rights reserved. MIT license.
```
Comment (eliassjogreen) [inline README.md:35]:
```suggestion
Copyright 2022-2023, the denosaurs team. All rights reserved. MIT license.
```

File: examples/hello_world.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { serve } from "https://deno.land/std@0.152.0/http/server.ts";
-import { router } from "https://deno.land/x/rutt/mod.ts";
```
Comment (eliassjogreen) [inline examples/hello_world.ts:2]: no, keep it this way, the import map fixes this for us that way the examples are kept the same as the actual code.

**Review by eliassjogreen — CHANGES_REQUESTED** [review]

**Comment by eliassjogreen** [comment]
Also fmt

File: README.md [hunk 4]
```diff
@@ -17,17 +17,19 @@ await serve(
 );
 ```
 
+## Projects using `rutt`
+
+- [denoland/fresh](https://github.com/denoland/fresh)
+
 ## Maintainers
 
 - Elias Sjögreen ([@eliassjogreen](https://github.com/eliassjogreen))
```
Comment (eliassjogreen) [inline README.md:26]: Want to be a maintainer?

**Review by eliassjogreen — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (load1n9) [reply README.md:26]: yes

**Pull request merged**